### DISCUSSION TITLE
📝Milestone 기능 사용 논의

### DISCUSSION THREAD

**Discussion by uush** [donghyun819/AutoCurtain_DY5th#31] [General]
# 논의 사항
Milestone은 discussion에 올린 것과 같이 기간을 설정하고 issue를 카운트 하여 진행상황을 퍼센트로 보여주는 기능입니다. 

저희 팀은 시작이 아니라 진행 중인 상황이라 이 기능을 써야할 지 고민이 됩니다.
의견을 적어 주세요!

**Comment by hyeonahhh** [comment]
저는 하는게 좋을 것 같아요!
